connectors/src/connectors/microsoft/temporal/client.ts

```diff
@@ -0,0 +1,15 @@
+import type { ModelId, Result } from "@dust-tt/types";
```

Comment:
Ok for you to split into the hierarchy we discussed?
```
connectors/microsoft/
-- lib/
-- sharepoint/
---- lib/
---- temporal/
-- teams/
---- lib/
---- temporal/
-- index.ts
```

Reply:
done